app/src/controllers/appInfo/selectors.js

```diff
@@ -58,3 +65,29 @@ export const instanceTypeSelector = (state) =>
   environmentSelector(state)[INSTANCE_TYPE] || NOT_PROVIDED;
 export const allowDeleteAccountSelector = (state) =>
   environmentSelector(state)[ALLOW_DELETE_ACCOUNT] === 'true';
+export const baseEventParametersSelector = createSelector(
```

Comment:
Let's reuse this selector at `app/src/components/main/analytics/AnalyticsWrapper.jsx` as well.